helper/helper.go

```diff
@@ -347,48 +355,69 @@ func DecryptSessionKey(
 }
 
 func encryptMessageArmored(key string, message *crypto.PlainMessage) (string, error) {
-	publicKeyRing, err := createPublicKeyRing(key)
+	ciphertext, err := encryptMessage(key, message)
+
 	if err != nil {
 		return "", err
 	}
 
-	pgpMessage, err := publicKeyRing.Encrypt(message, nil)
+	ciphertextArmored, err := ciphertext.GetArmored()
+
 	if err != nil {
-		return "", errors.Wrap(err, "gopenpgp: unable to encrypt message")
+		return "", errors.Wrap(err, "gopenpgp: unable to armor ciphertext")
+	}
+
+	return ciphertextArmored, nil
+}
+
+func decryptMessageArmored(privateKey string, passphrase []byte, ciphertextArmored string) (*crypto.PlainMessage, error) {
+	ciphertext, err := crypto.NewPGPMessageFromArmored(ciphertextArmored)
+	if err != nil {
+		return nil, errors.Wrap(err, "gopenpgp: unable to parse ciphertext")
 	}
 
-	ciphertext, err := pgpMessage.GetArmored()
+	return decryptMessage(privateKey, passphrase, ciphertext)
+}
+
+func encryptMessage(key string, message *crypto.PlainMessage) (*crypto.PGPMessage, error) {
+	publicKeyRing, err := createPublicKeyRing(key)
+
 	if err != nil {
-		return "", errors.Wrap(err, "gopenpgp: unable to armor message")
+		return nil, err
+	}
+
+	ciphertext, err := publicKeyRing.Encrypt(message, nil)
+
+	if err != nil {
+		return nil, errors.Wrap(err, "gopenpgp: unable to encrypt message")
```

Comment:
This is a bit nitpicky but you added newlines before the `if err != nil` in a bunch of places in this PR, could you keep the error handling next to the line that it's.. "handling"? ^.^